package.json

```diff
@@ -22,28 +22,27 @@
   "publishConfig": {
     "access": "public"
   },
+  "workspaces": [
+    "demo"
+  ],
```

Comment:
Previously the demo app used axe-storybook-testing through some `link`-ing shenanigans.

Now it uses workspaces, which really simplifies the hole procedure.

Note that Yarn could have also done this, so this isn't a benefit of using npm.